src/actions/adduser.js

```diff
@@ -30,8 +30,8 @@ module.exports = async function(env, options) {
     }
 
     try {
-        await app.userDb.addUser(username, password, userIsAdmin);
-        console.log(`Added new user ${username}`);
+        const user = await app.userDb.addUser(username, password, userIsAdmin);
+        console.log(`Added new user ${user.username}`);
```

Comment:
This should use the `l.info()` logger

Reply:
I think this using console.log is intended, as setting log level to debug would prevent result of adding the user from being shown

Reply:
OH, of course, yes. This is in the action. All good